modules/fb_instant_articles_api_rules/fb_instant_articles_api_rules.info

```diff
@@ -0,0 +1,5 @@
+name = Facebook Instant Articles API Rules
+description = Provide Drupal rules to manage facebook instant article using API.
+package = Facebook Instant Articles
+core = 7.x
+dependencies[] = fbia
```

Comment:
Updated the dependency now and this module can be used without `rules` module as well since we have a `hook_action_info` so which can be used with default `action` and `trigger` module.